src/test/java/net/greatstart/services/ProjectServiceTest.java

```diff
@@ -0,0 +1,68 @@
+package net.greatstart.services;
+
+import net.greatstart.Main;
+import net.greatstart.dao.ProjectDao;
+import net.greatstart.model.Project;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static org.mockito.Mockito.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = Main.class)
+public class ProjectServiceTest {
+    @Mock
+    ProjectDao mockProjectDao;
+    ProjectService projectService;
+    Project project = new Project();
+
+    @Before
```

Comment:
You do not need this. You can run test suite with MockitoJUnitRunner.class, and init tested class with @InjectMocks annotation